tests/phpunit/CRM/Contribute/BAO/ContributionTest.php

```diff
@@ -783,4 +783,52 @@ protected function addParticipantWithContribution() {
     return array($lineItems, $contributions);
   }
 
+  /**
+   * checkLineItems() check if total amount matches the sum of line total
```

Comment:
Trivial point - the full comment standard (which we are encouraging but not enforcing) has a capital letter at the start & a full stop at the end